lib/Page.js

```diff
@@ -243,7 +243,15 @@ Page.prototype.collectHeadingsInContent = function (content) {
     $('modal').remove();
     $('panel').remove();
     $(headingsSelector).each((i, heading) => {
-      this.headings[$(heading).attr('id')] = $(heading).text();
+      const headingId = $(heading).attr('id');
+      const keywords = [];
+      $(heading).next().children('keyword').each((index, element) => {
```

Comment:
This isn't general enough for the use case in the target issue. Keywords don't always come directly after a heading in a single `<p>` element.